### PR TITLE
feat: --destination option

### PR DESCRIPTION
**Pull request by martin-minarik**
Resolves issue #23

1. Added **"--destination"** option to **parser**.
2. Added a **"destination"** parameter to the **"download(...)"**
    and **"download_video_audio(...)"** functions located in the **"download.py"**.
4. Passed destination to **"pytube.YouTube.download(...)"** located in **"download_video_audio(...)"**.

=)

### CODE REVIEW

File: youtube_bz/main.py [hunk 1]
```diff
@@ -20,6 +20,7 @@ async def run_command(args: argparse.Namespace):
         await commands.download(
             args.mbid,
             verbose,
+            destination=args.destination
```
Comment (Flowrey) [inline youtube_bz/main.py]: `args.destination` is explicit enough i don't think we need to use a named parameter here.

File: youtube_bz/main.py [hunk 2]
```diff
@@ -43,6 +44,8 @@ def get_command_parser() -> argparse.ArgumentParser:
     download_parser.add_argument("mbid", help="music brainz identifer of a release")
     download_parser.add_argument("--verbose", action="store_true")
 
+    download_parser.add_argument("--destination", help="Path to the output directory")
```
Comment (Flowrey) [inline youtube_bz/main.py]: Maybe we can add a short flag like `-d` too.
e. g `download_parser.add_argument("-d", "--destination", help='Path to the output directory")`

**Review by Flowrey — CHANGES_REQUESTED** [review]
Thanks for contributing, seems good to me, maybe you can just add a short flag `-d` and remove the named parameters in `commands.download`.

File: youtube_bz/main.py (hunk 2 above)
Comment (martin-minarik) [reply youtube_bz/main.py]: Sure thing!

**Comment by Flowrey** [comment]
Thanks for the change, just one last thing the CI doesn't pass, you just need to run `black` and `isort --profile=black` on your code, and i will merge.

**Review by Flowrey — APPROVED** [review]

**Comment by Flowrey** [comment]
You forgot to run `isort --profile=black .` 😜

**Comment by martin-minarik** [comment]
I see that it quite depends on which working directory it's being called from.

I tried calling it from youtube_bz/commands and it seemed to work fine the first time. （°々°）

**Pull request merged**